docs/content/integrations/dbt/reference.mdx

```diff
@@ -123,6 +123,81 @@ In production, `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD` should be unset so that your
 
 ---
 
+## Using config with `@dbt_assets`
```

Comment:
This should probably go above the `Scheduling jobs`. Otherwise, we're introducing this section in the middle of other sections that deal with deploying a Dagster & dbt project.